PaddleHQ/transaction-search
# TOOLS-248: Update project template

### Ticket(s):
Link to ticket:
https://paddle.atlassian.net/browse/TOOLS-248

### Checklist

### What has been done:
- Update `python-microservice-template` with names specific to this project
- Add README.md

### How to test:
It's been tested on `development`, so I suggest to test it direcly on `staging`.
- Create, using `deployment-user.cfn.yml`, a user called `paddle-{env}-transaction-search-deploy` and save AWS (programmatic access) credentials
- Create a user with the same name on Jenkins (both ID and description should match the naming pattern) and add the AWS credentials previously generated
- Deploy through Jenkins
- Checks:
- [x] Deployment process succeeded
- [x] `paddle-{env}-transaction-search` S3 bucket has been created
- [x] `paddle-{env}-transaction-search-fetch` lambda works
- [ ] `Codecov` correctly reports coverage on Github
- [x] Deploy messages are correctly sent to `#blackstar-releases`
